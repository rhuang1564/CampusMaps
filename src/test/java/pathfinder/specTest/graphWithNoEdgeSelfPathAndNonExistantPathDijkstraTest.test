# Find a path between node and another node 
# and node to itself in Dijkstra's algorithm
# in a no edged graph

CreateGraph graph

AddNode graph A
AddNode graph B

FindPath graph A B

FindPath graph A A