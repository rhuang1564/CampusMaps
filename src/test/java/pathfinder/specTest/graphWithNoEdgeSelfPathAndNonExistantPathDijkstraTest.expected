# Find a path between node and another node 
# and node to itself in Dijkstra's algorithm
# in a no edged graph

created graph graph

added node A to graph
added node B to graph

path from A to B:
no path found

path from A to A:
total cost: 0.000